extensions/android-adb-input/src/index.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Action,
+  ActionPanel,
+  Detail,
+  Form,
+  getPreferenceValues,
+  popToRoot,
+  PreferenceValues,
+  showHUD,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import { useState, useEffect } from "react";
+import { execaCommand } from "execa";
+import * as fs from "fs";
+
+type Error = {
+  title: string;
+  message: string;
+};
+
+function getAdbDir(): string {
+  const preferences = getPreferenceValues<PreferenceValues>();
+  const adbDir = preferences["adbDir"];
+  if (adbDir) {
+    return adbDir;
+  }
+
+  const home = process.env.HOME;
+  return `${home}/Library/Android/sdk/platform-tools`;
+}
+
+function existsAdb(): boolean {
+  return fs.existsSync(`${getAdbDir()}/adb`);
+}
+
+async function getDeives(): Promise<string[]> {
```

Comment:
```suggestion
async function getDevices(): Promise<string[]> {
```
Totally NIT but could be nice to have corrected 😄

Reply:
Fix [a5f1062](https://github.com/raycast/extensions/pull/964/commits/a5f106229b7eee34278b237abe5ce7f24fd4aa8b)